maps/sccv_horizon/sccv_horizon-2_deck_2.dmm

```diff
@@ -23854,7 +23854,8 @@
 	},
 /obj/item/stamp{
 	pixel_x = -5;
-	pixel_y = 13
+	pixel_y = 13;
+	name = "Operation Manager's stamp"
```

Comment:
I know this is how the QM's stamp worked - But we should make it its own subtype because then it will show up on the chameleon stamp.

Reply:
Changes made.